tests/integration/dash_assets/test_assets.py

```diff
@@ -0,0 +1,150 @@
+import json
+import time
+import itertools
+
+import dash_html_components as html
+import dash_core_components as dcc
+
+from dash import Dash
+from tests.integration.IntegrationTests import IntegrationTests
+from tests.integration.utils import wait_for, invincible
+
+
+class TestAssets(IntegrationTests):
+
+    def setUp(self):
+        def wait_for_element_by_id(id_):
+            wait_for(lambda: None is not invincible(
+                lambda: self.driver.find_element_by_id(id_)
+            ))
+            return self.driver.find_element_by_id(id_)
+        self.wait_for_element_by_id = wait_for_element_by_id
```

Comment:
Why the closure pattern here? Can't this just be part of `IntegrationTests`?

Reply:
I didn't check the details, it was a simple 🚚  

In general all these wait_for needs a better refactoring